### DISCUSSION TITLE
Syntax for not equal ?

### DISCUSSION THREAD

**Discussion by JLD95** [maartentamboer/X-Touch-Mini-FS2020#106] [Q&A] [answered]
Hi,

I would like to switch on a LED if Weather radar is on 1 or 2 position.

At now I test only 1 position:
"mobiflightsimvar_led": "(L:XMLVAR_A320_WEATHERRADAR_SYS) 1 == if"

but it will be fine if I can replace == by <> ? or another syntax  ?

**Comment by Koseng** [comment, marked as answer]
Comparison operators
https://docs.flightsimulator.com/html/Additional_Information/Reverse_Polish_Notation.htm

**Reply by JLD95** [reply]
Thanks
